src/test/java/Study/SetTest.java

```diff
@@ -0,0 +1,58 @@
+package Study;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.*;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    @DisplayName("TestSet size")
+    void testSetSize(){
+        assertThat(numbers.size()).isEqualTo(3);
+    }
+
+
+    @Test
+    @DisplayName("TestSet Contains")
+    void testContains(){
+        assertThat(numbers.contains(1)).isTrue();
+        assertThat(numbers.contains(2)).isTrue();
+        assertThat(numbers.contains(3)).isTrue();
```

Comment:
```suggestion
        assertThat(numbers).containsExactlyInAnyOrder("1", "2", "3")
```

이 함수를 한번 활용해보는 것도 좋을 것 같아요 :)

Reply:
추가로 다양한 assertion을 테스트하는 상황이라면 junit의 assertAll을 한번 학습해보시는 것을 추천드릴게요 👍🏼